src/osd/scrubber/pg_scrubber.h

```diff
@@ -415,6 +418,13 @@ class PgScrubber : public ScrubPgIF, public ScrubMachineListener {
 
   [[nodiscard]] bool was_epoch_changed() const final;
 
+  void set_being_scrubbed() final;
+  void clear_being_scrubbed() final;
+
+  void set_finishing_flag() final { m_finish_sequence_started = true; }
+  void clear_finishing_flag() final { m_finish_sequence_started = false; }
+  bool is_finishing_flag_set() const final { return m_finish_sequence_started; }
```

Comment:
I don't really understand what's going on here.

Reply:
removed all that 'finishing flag' stuff.